react-components/src/desktop/messages.js

```diff
@@ -30,6 +30,11 @@ var intlData = {
         emailSupportTooltip: "support@cyverse.org",
         supportChatLink: "Chat With Support",
         supportChatTooltip: "Chat may be disabled by your browser's settings",
+        collaboratorsTooltip: "Manage your private set of collaborators",
+        teamsTooltip:
+            "Manage public or private groups of collaborators. Collaborators in a group can all share resources within the group",
```

Comment:
It might be better to stick with the `Team` terminology here:
```suggestion
            "Manage public or private groups of collaborators. Collaborators in a team can all share resources with that team.",
```